kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -21,7 +21,7 @@
 
 ;;; dynamic variables to control function
 
-(defonce ^:dynamic *select-type* select-default)
+(defonce ^:dynamic *select-type* select-distinct)
```

Comment:
Whoops; I think it’s not needed.  That’s a relic from when I was trying to write ice rules, and is not used.  Sorry!

 

From: Marc Daya <notifications@github.com>
Reply-To: UCDenver-ccp/kr <reply@reply.github.com>
Date: Tuesday, May 31, 2016 at 3:27 PM
To: UCDenver-ccp/kr <kr@noreply.github.com>
Cc: Elizabeth White <elizabeth.white@raritanfarm.com>, Author <author@noreply.github.com>
Subject: Re: [UCDenver-ccp/kr] More talking (#4)

 

In kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj:
> @@ -21,7 +21,7 @@
>  
>  ;;; dynamic variables to control function
>  
> -(defonce ^:dynamic *select-type* select-default)
> +(defonce ^:dynamic *select-type* select-distinct)
What is the motivation for changing the default?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.